pkg/rbac/userData.go

```diff
@@ -17,17 +17,24 @@ import (
 
 // Contains data about the resources the user is allowed to access.
 type userData struct {
-	// clusters     []string              // Managed clusters where the user has view access.
 	csResources []resource            // Cluster-scoped resources on hub the user has list access.
 	nsResources map[string][]resource // Namespaced resources on hub the user has list access.
+	clusters    []string              // Managed clusters where the user has view access.
 
+	clustersErr error // Error while updating clusters data.
+	// clustersLock      sync.Mutex // Locks when clusters data is being updated. NOTE: not implmented because we use the nsrLock
+	clustersUpdatedAt time.Time // Time clusters was last updated.
+
+	// Internal fields to manage the cache.
```

Comment:
Move comment.
```suggestion
	// Internal fields to manage the cache.
	clustersErr error // Error while updating clusters data.
	// clustersLock      sync.Mutex // Locks when clusters data is being updated. NOTE: not implmented because we use the nsrLock
	clustersUpdatedAt time.Time // Time clusters was last updated.
```